<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mesero</class>
 <widget class="QDialog" name="mesero">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>151</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: #C0C4CF;
border-right: 1px solid;

}
</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Img.qrc">:/img/Iconos/logosinfondo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Salir">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  border:0px solid gray;
  padding: 0 8px;
  background: yellow;
background-color: rgba(216, 223, 233,0);
}
QPushButton:hover
{
	background-color: rgb(223, 138, 19);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="Img.qrc">
      <normaloff>:/img/Iconos/salida.png</normaloff>:/img/Iconos/salida.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Buscar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  border:0px solid gray;
  padding: 0 8px;
  background: yellow;
background-color: rgba(216, 223, 233,0);
}
QPushButton:hover
{
	background-color: rgb(223, 138, 19);
}</string>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
    <property name="icon">
     <iconset resource="Img.qrc">
      <normaloff>:/img/Iconos/lupa.png</normaloff>:/img/Iconos/lupa.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="NuevoPedido">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  border:0px solid gray;
  padding: 0 8px;
  background: yellow;
background-color: rgba(216, 223, 233,0);
}
QPushButton:hover
{
	background-color: rgb(223, 138, 19);
}</string>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
    <property name="icon">
     <iconset resource="Img.qrc">
      <normaloff>:/img/Iconos/anadir.png</normaloff>:/img/Iconos/anadir.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="NuevoPedido_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
  border:0px solid gray;
  padding: 0 8px;
  background: yellow;
background-color: rgba(216, 223, 233,0);
}
QPushButton:hover
{
	background-color: rgb(223, 138, 19);
}</string>
    </property>
    <property name="text">
     <string>Ordenes</string>
    </property>
    <property name="icon">
     <iconset resource="Img.qrc">
      <normaloff>:/img/Iconos/terminado.png</normaloff>:/img/Iconos/terminado.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>681</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #faf4ee;</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>18</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Buscar Platillos</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>621</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 1px solid; 
border-radius: 10px;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>ID PLATILLO</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="TablaPlatillos">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>641</width>
       <height>341</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QListWidget" name="listamenu">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>251</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listapedido">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>256</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="bebidas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>256</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="postres">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>256</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Desayunos</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Bebidas</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>320</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Postres</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>65</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Orden</string>
     </property>
    </widget>
    <widget class="QPushButton" name="CrearPedido">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>410</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
  border:0px solid gray;
  padding: 0 8px;
  background: yellow;
background-color: rgba(216, 223, 233,0);
}
QPushButton:hover
{
	background-color: rgb(223, 138, 19);
}</string>
     </property>
     <property name="text">
      <string>Crear Pedido</string>
     </property>
     <property name="icon">
      <iconset resource="Img.qrc">
       <normaloff>:/img/Iconos/iconfinder_My_order_6457548.png</normaloff>:/img/Iconos/iconfinder_My_order_6457548.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>420</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Importe: </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="total">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>420</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="mesa">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;
</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>No. Mesa:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>641</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>-120</x>
       <y>360</y>
       <width>221</width>
       <height>151</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Img.qrc">:/img/Iconos/platillos-png.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Caviar Dreams</family>
       <pointsize>22</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Ordenes</string>
     </property>
    </widget>
    <zorder>label_9</zorder>
    <zorder>tableWidget</zorder>
    <zorder>label_10</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Img.qrc"/>
 </resources>
 <connections/>
</ui>
